content/agent/faq/agent-configuration-file-migration.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Agent configuration file migration
+kind: faq
+further_reading:
+---
+
+[If you haven't done it already, upgrade your Agent v5 to Agent v6][1]
+
+To automatically transition Agent configuration paths and formats from Agent v5 to Agent v6, use the `import` command. The command parses an existing v5 `datadog.conf` and converts the configuration options to the new v6 `datadog.yaml` format. It also copies configuration files for checks that are currently enabled.
+
+{{< tabs >}}
+{{% tab "Linux" %}}
+
+`sudo -u dd-agent -- datadog-agent import`
```

Comment:
the correct command on Linux is `sudo -u dd-agent -- datadog-agent import /etc/dd-agent/ /etc/datadog-agent/`